src/backend/launcher.ts

```diff
@@ -674,9 +674,16 @@ async function prepareWineLaunch(
       writeFileSync(appsNamesPath, JSON.stringify([appName]), 'utf-8')
       hasUpdated = true
     } else {
-      const installedGames: string[] = JSON.parse(
-        readFileSync(appsNamesPath, 'utf-8')
-      )
+      let installedGames: string[] = []
+
+      try {
+        installedGames = JSON.parse(
+          readFileSync(appsNamesPath, 'utf-8')
+        ) as string[]
+      } catch (error) {
+        logError(error)
```

Comment:
Do we really want to just continue on here? My thinking is
- user has many different games sharing the same prefix
- `installed_games` file gets corrupted somehow, perhaps in a way that's fixable with some inspection
- Heroic overwrites `installed_games`, removing all previously-recorded AppNames

Reply:
yeah, that's a good point, I was thinking that most of the times users have a prefix per game so it's kinda safe to just regenerate the file to not prevent launching the game, but I guess a better error message to fix the file can also help us understand over time why it can even get corrupted

Reply:
Btw do you guys also emply AI to review the small changes or is it done just manually always?

Reply:
I don't use AI in general

Reply:
Ah ok nice, was just trying to understand the contribution process and review process. As some open source projects use AI bots for initial reviews these days.